src/main/java/frc/robot/commands/CmdDriveWithJoystick.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.drive.Drive;
+
+public class CmdDriveWithJoystick extends CommandBase
+{
+    DoubleSupplier _xInput;// joystick axis for x/strafe
+    DoubleSupplier _yInput;// joystick axis for y/drive
+    DoubleSupplier _rInput;// joystick axis for rotation
+    BooleanSupplier _robotCentricInput;// button for field/robot centered driving
```

Comment:
Let's align variable names and comments. It'll look a little nicer 😄 

```suggestion
    DoubleSupplier  _xInput;               // joystick axis for x/strafe
    DoubleSupplier  _yInput;               // joystick axis for y/drive
    DoubleSupplier  _rInput;               // joystick axis for rotation
    BooleanSupplier _robotCentricInput;    // button for field/robot centered driving
```